src/components/header/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Button, Flex, Image, Stack, Menu, MenuButton, MenuList, MenuItem, Avatar, Icon } from '@chakra-ui/react';
+
+import { BellIcon, SearchIcon } from '@chakra-ui/icons';
+import logo from '@/assets/images/logo.png';
+import defaultProfile from '@/assets/images/musseuk_hood.png';
+import InputField from '@/components/Header/components/Input';
+import { useNavigate } from 'react-router-dom';
+
+type HeaderProps = {
+  profileImg?: string;
+  userId: string;
+};
+
+const Header = ({ profileImg, userId }: HeaderProps) => {
+  const navigate = useNavigate();
+
+  const handleMenuItem = (path: string) => {
+    navigate(path);
+  };
+  return (
+    <Box h={12} px={3}>
+      <Flex gap={4} alignItems={'center'} h="100%" justifyContent={'space-between'}>
+        <Image h={8} cursor="pointer" src={logo} alt="logo" onClick={() => handleMenuItem('/')} />
+        <InputField
+          icon={<Icon as={SearchIcon} />}
+          mw={'35rem'}
+          inputProps={{
+            placeholder: '사용자 이름을 입력해 주세요',
+            size: 'sm',
+            fontSize: '14px',
+            fontWeight: 'bold'
+          }}
+        />
+        <Stack direction={'row'} spacing={3}>
+          <BellIcon w={6} h={6} cursor={'pointer'} />
+          <Menu autoSelect={false}>
+            <MenuButton as={Button} rounded={'full'} variant={'link'} cursor={'pointer'} minW={0}>
+              <Avatar size={'xs'} src={profileImg ?? defaultProfile} />
+            </MenuButton>
+            <MenuList minW={36} p={0}>
+              <MenuItem
+                borderTopLeftRadius={5}
+                borderTopRightRadius={5}
+                h={7}
+                justifyContent={'center'}
+                _hover={{ bg: 'var(--gray01)' }}
+                onClick={() => handleMenuItem(`/profile/${userId}`)}>
+                프로필
+              </MenuItem>
+              <MenuItem
+                borderBottomLeftRadius={5}
+                borderBottomRightRadius={5}
+                h={7}
+                justifyContent={'center'}
+                _hover={{ bg: 'var(--gray01)' }}
+                onClick={() => handleMenuItem(`/changePassword/${userId}`)}>
```

Comment:
P1:
앗! 비밀번호 변경 페이지의 path에 대한 이야기를 아직 나누지 않았었군요! 😮

비밀번호 변경은 현재 로그인된 사용자의 비밀번호만 변경되는 것이고 다른 사용자의 비밀번호를 변경할 수는 없으니 userId 파라미터를 보내지 않아도 좋아보여요!

```suggestion
                onClick={() => handleMenuItem(`/changePassword`)}>
```

예를 들어서 frong123 라는 사용자가 `/changePassword/pororo` 라는 path에 들어와서 pororo 유저의 비밀번호를 변경할 수는 없는 것 처럼요!

Reply:
넵!! 수정하겠습니다🙂